pages/[planetName].tsx

```diff
@@ -4,17 +4,18 @@ import FactContainer from '../components/funFacts';
 import Moons from '../components/moon';
 import lowerCase from '@/lib/lowercaseWord';
 import { planet } from '@/lib/types';
+import Loading from '@/components/loading';
 
 export default function Planets() {
   const router = useRouter();
   const { planetName } = router.query;
   const [planetData, setPlanetData] = useState<planet>();
   const [core, setCore] = useState<boolean>(false);
```

Comment:
For consistency, set this useState to true and then remove from line 18, yeah?

Reply:
Removed from this file just like on the moons.tsx component

![image](https://user-images.githubusercontent.com/109567499/229324809-7f7693d8-1d85-45ce-b21b-71afa4121ff5.png)